core/src/main/AndroidManifest.xml

```diff
@@ -51,7 +52,7 @@
     <provider
       android:name=".reader.ZimContentProvider"
       android:authorities="${applicationId}.zim.base"
-      android:exported="true" />
```

Comment:
An error message popped up stating there was a security risk in keeping exported="true".  Happy to drop it if needed.

Reply:
I wouldn't change this for now, it could have ramifications. The security risk is somebody being able to read zim files in another app, so I am okay with it